calico-cloud/_includes/release-notes/_v3.14.0-pre-release-notes.mdx

```diff
@@ -19,7 +19,7 @@
 - In BPF dataplane mode, Felix now handles single-block IPAM pools. Previously single-block pools resulted in a collision when programming the dataplane routes. [felix #2245](https://github.com/projectcalico/felix/pull/2245) (@fasaxc)
 - None required [felix #2233](https://github.com/projectcalico/felix/pull/2233) (@tomastigera)
 - None required [felix #2232](https://github.com/projectcalico/felix/pull/2232) (@tomastigera)
-- [Openstack] Allow DHCP from the workload, on kernels where rp_filter doesn't already [felix #2231](https://github.com/projectcalico/felix/pull/2231) (@neiljerram)
```

Comment:
This correction is based on the author of https://github.com/projectcalico/felix/pull/2231 -- both this incorrect spelling and the correct spelling appeared in the output I was reviewing, and a quick spot check determined this was the incorrect spelling.